### ISSUE TITLE
When will  model MaskFeat available

### ISSUE THREAD

**Issue by RechelTeamo** [facebookresearch/pytorchvideo#163]
Recently I read this paper： Masked Feature Prediction for Self-Supervised Visual Pre-Training。
The idea of this article is very new, so I want to try it, when will you open the source code?
Thank you

**Comment by kalyanvasudev** [comment]
Yes, we do intend to release the framework. @haooooooqi , do you have timeline for this?

**Comment by haofanwang** [comment]
Any update?

**Comment by BlockchainRev** [comment]
Any updates?

**Comment by RechelTeamo** [comment]
I reproduced a version myself, although the top-1 accuracy is not as good as the results in the paper.

**Comment by gittygupta** [comment]
@haooooooqi any update on this?

**Comment by SnowdenLee** [comment]
@RechelTeamo  Hi, could you maybe share the codes? Thanks :)

**Comment by TinnyFlames** [comment]
@RechelTeamo  Hi! I also tried to implement the Maskfeat method. Could you share the code or give us some hints? I stuck with the dis-match between image size (224,224,3) and hog feature size (14,14,108) when masking out image patches. Do you have any suggestions?  Thanks (●'◡'●)

**Comment by RechelTeamo** [comment]
@TinnyFlames I gauss you are from china. Maybe you can send me your WeChat or E-mail to tyzheng@sjtu.edu.cn .
Then we can have a talk.

**Comment by Wallace-222** [comment]
Any updates?

**Comment by irsLu** [comment]
Any updates?

**Comment by linhanwang** [comment]
Any updates?
